### PR TITLE
add error message to postcode field

### LINKED ISSUE #1700: Should an error message be removed if the user enters the valid postal code
## Summary

Error message should be removed if the user enters the valid postal code

## Steps to reproduce
Go to location page
Enter invalid postal code 123456 -> Error message is displayed
Enter the correct postal code --> Error message is still displayed

postalCode

## Unresolved questions

> Are there any related issues you consider out of scope for this issue that could be addressed in the future?

### LINKED ISSUE #1745: Error Message is not set on postal code field
## Summary

We need to set an Error Message on the postal code field.

Error message is linked to the error summary, we need to have a connection between them for screen readers. 

## Steps to reproduce

1. Trigger a validation error on postal code field
image

## Design details
Location

### PR DESCRIPTION
**Pull request by DianeLiu2019**
Fixes #1745 
Fixes #1700 (issue)

# Description
set an Error Message on the postal code field.

# Checklist:

- [ x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)

### CODE REVIEW

File: f2/src/forms/LocationInfoForm.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const validate = values => {
     ).test(values.postalCode)
   ) {
     errors.postalCode = 'locationInfoForm.warning'
+    errors.postalCodelabel = 'locationInfoForm.labelWarning'
```
Comment (amazingphilippe) [inline f2/src/forms/LocationInfoForm.js]: No need to add a new error! You can remove this line, and its translations in locales folder.

File: f2/src/forms/LocationInfoForm.js [hunk 2]
```diff
@@ -103,6 +104,7 @@ export const LocationInfoForm = ({ onSubmit }) => {
             <Field
               name="postalCode"
               label={<Trans id="locationinfoPage.postalCode" />}
+              errorMessage={<Trans id="locationInfoForm.labelWarning" />}
```
Comment (amazingphilippe) [inline f2/src/forms/LocationInfoForm.js]: simply use `locationInfoForm.warning` id! You can reuse the same translation.

Comment (amazingphilippe) [reply f2/src/forms/LocationInfoForm.js]: Also, this means you can remove lines 54 to 58! the `{false ? ( // mark ids for lingui` section

Comment (DianeLiu2019) [reply f2/src/forms/LocationInfoForm.js]: fixed. Thank you

File: f2/src/forms/LocationInfoForm.js (hunk 1 above)
Comment (DianeLiu2019) [reply f2/src/forms/LocationInfoForm.js]: okay. reuse the same error message

**Pull request merged**